hw/ip/power_manager/data/power_manager.hjson

```diff
@@ -8,13 +8,13 @@
   ],
   regwidth: "32",
   registers: [
-    { name:     "POWER_GATE_CORE",
-      desc:     "Used to power gate the core",
+    { name:     "POWER_GATE_DOMAIN",
+      desc:     "Used to power gate domain",
       resval:   "0x00000000"
       swaccess: "rw",
       hwaccess: "hro",
       fields: [
-        { bits: "0", name: "POWER_GATE_CORE", desc: "Power Gate Core Reg" }
+        { bits: "31:0", name: "POWER_GATE_DOMAIN", desc: "Power Gate Domain Reg" }
```

Comment:
in my opinion, it is safer to have 1bit per register per power domain, it simplifies SW as you do not need shift or etc. Plus it can be easily generated

Reply:
Fixed.